packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -592,6 +592,56 @@ void main() {
       );
     });
 
+    testWidgets('width of wheel in background does not increase at large widths', (WidgetTester tester) async {
+
+      final Widget dateWidget = CupertinoDatePicker(
+        mode: CupertinoDatePickerMode.date,
+        onDateTimeChanged: (_) { },
+        initialDateTime: DateTime(2018, 1, 1, 10, 30),
+      );
+
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoPageScaffold(
+            child: Center(
+              child: SizedBox(
+                height: 200.0,
+                width: 300.0,
+                child: dateWidget,
+              ),
+            ),
+          ),
+        ),
+      );
+
+      double decemberX = tester.getBottomLeft(find.text('December')).dx;
+      double octoberX = tester.getBottomLeft(find.text('October')).dx;
+      final double distance = octoberX - decemberX;
+
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoPageScaffold(
+            child: Center(
+              child: SizedBox(
+                height: 200.0,
+                width: 3000.0,
+                child: dateWidget,
+              ),
+            ),
+          ),
+        ),
+      );
+
+      decemberX = tester.getBottomLeft(find.text('December')).dx;
+      octoberX = tester.getBottomLeft(find.text('October')).dx;
+
+      // The wheel does not bend outwards at large widths.
+      expect(
+        distance >= (octoberX - decemberX),
```

Comment:
This test didn't actually work because of a bug - fixed it here https://github.com/flutter/flutter/pull/121342/files#r1148615910